File: Course_01/Java Language/Vetores/mais_velho.java
package Vetores;

import java.util.Locale;
import java.util.Scanner;

public class mais_velho {

	public static void main(String[] args) {
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		int i, n, maisVelho, iMaisVelho;
		
		System.out.print("Quantas pessoas você vai digitar? ");
		n = sc.nextInt();
		
		String [] nomes = new String [n];
		int [] idade = new int [n];
		
		for (i = 0; i < n; i++) {
			System.out.println("Dados da " + (i + 1) + "a pessoa:");
			System.out.print("Nome: ");
			sc.nextLine();
			nomes[i] = sc.nextLine();
			System.out.print("Idade: ");
			idade[i] = sc.nextInt();
		}
		
		
		maisVelho = idade[0];
		iMaisVelho = 0;
			
		for (i = 0; i < n; i++) {
			if (maisVelho < idade[i]) {
				maisVelho = idade[i];
				iMaisVelho = i;
			}
		}
		
		System.out.print("PESSOA MAIS VELHA: " + nomes[iMaisVelho]);

		sc.close();
	}

}

File: Course_01/Python/Vetores/mais_velho.py
n = int(input("Quantas pessoas você vai digitar? "))

nome = [0 for x in range(n)]
idade = [0 for x in range(n)]

for i in range(0, n):
    print(f"Dados da {(i + 1)}a pessoa:")
    nome[i] = input("Nome: ")
    idade[i] = int(input("Idade: "))
print("Pessoa mais velha: ", end="")
maisVelho = idade[0]
nomeMaisVelho = nome[0]
for i in range(0, n):
    if maisVelho < idade[i]:
        maisVelho = idade[i]
        nomeMaisVelho = nome[i]
print(nomeMaisVelho)